File: Easy/Java/2180. Count Integers With Even Digit Sum.java
class Solution {
    public int countEven(int num) {
        int count = 0;

        for(int i = 2; i <= num; i++){
            int temp = i;
            int sum = 0;
            while(0 < temp){
                sum += temp % 10;
                temp /= 10;
            }
            if(sum % 2 == 0)
                count++;
        }

        return count;
    }
}

File: Easy/Python/2180. Count Integers With Even Digit Sum.py
class Solution(object):
    def countEven(self, num):
        count = 0

        for i in range(2, num + 1):
            temp = i
            this_sum = 0
            while 0 < temp:
                this_sum += temp % 10
                temp /= 10
            if this_sum % 2 == 0:
                count += 1

        return count
